Dockerfile.test

```diff
@@ -0,0 +1,58 @@
+# Dockerfile for testing swarm
+# How to use:
+#   docker build --rm --force-rm -f Dockerfile.test -t swarm-test .
+#   docker run --rm -it --privileged swarm-test script/testall
+FROM debian:jessie
```

Comment:
How about starting from `aluzzardi/docker` so it comes pre-installed with docker and no need to install its dependencies?